code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -50,13 +50,24 @@
 /datum/chemical_reaction/plasmasolidification
 	name = "Solid Plasma"
 	id = "solidplasma"
-	required_reagents = list("iron" = 5, "frostoil" = 5, "plasma" = 20)
+	required_reagents = list("frostoil" = 5, "plasma" = 20)
```

Comment:
Better question is why the person is drinking that much plasma, and it would only stop like 1/3rd of the sting